### PR TITLE
Add response for GH action slash command

### LINKED ISSUE #135: Have slash-command-dispatch respond with link to the action url
This would be a nice QOL improvement.
It will respond with the job after it's completed.

https://github.com/peter-evans/slash-command-dispatch#responding-to-the-comment-on-command-completion

```yaml
      - name: Create URL to the run output
        id: vars
        run: echo "run-url=https://github.com/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID" >> $GITHUB_OUTPUT

      - name: Create comment
        uses: peter-evans/create-or-update-comment@v2
        with:
          token: ${{ secrets.PAT }}
          repository: ${{ github.event.client_payload.github.payload.repository.full_name }}
          issue-number: ${{ github.event.client_payload.github.payload.issue.number }}
          body: |
            [Command run output][1]

            [1]: ${{ steps.vars.outputs.run-url }}
```

### PR DESCRIPTION
**Pull request by zack-is-cool**
Closes #135

### CODE REVIEW

**Comment by zack-is-cool** [comment]
Ideally, we would have some kind of org-level bot account do this instead of having a specific user's pat.

**Comment by ntwkninja** [comment]
@RothAndrew any thoughts here?

File: .github/workflows/test-command.yml [hunk 1]
```diff
@@ -70,6 +70,22 @@ jobs:
     needs: parse
     if: needs.parse.outputs.run-ping == 'true'
     steps:
+      - name: Create URL to the run output
+        id: vars
+        run: echo "run-url=https://github.com/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID" >> $GITHUB_OUTPUT
+
+      #Will edit the comment that triggered the /test comment and add the run-url
+      - name: Create comment
```
Comment (RothAndrew) [inline .github/workflows/test-command.yml:78]:
```suggestion
      - name: Update comment
```

File: .github/workflows/test-command.yml [hunk 2]
```diff
@@ -70,6 +70,22 @@ jobs:
     needs: parse
     if: needs.parse.outputs.run-ping == 'true'
     steps:
+      - name: Create URL to the run output
+        id: vars
+        run: echo "run-url=https://github.com/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID" >> $GITHUB_OUTPUT
+
+      #Will edit the comment that triggered the /test comment and add the run-url
```
Comment (RothAndrew) [inline .github/workflows/test-command.yml:77]:
```suggestion
      #Will update the comment that triggered the /test comment and add the run-url
```

File: .github/workflows/test-command.yml [hunk 3]
```diff
@@ -105,6 +121,22 @@ jobs:
           GITHUB_REF: ${{ github.event.client_payload.pull_request.head.ref || github.ref_name }}
           GITHUB_OWNER: ${{ github.event.client_payload.github.payload.repository.owner.login || github.repository_owner }}
 
+      - name: Create URL to the run output
+        id: vars
+        run: echo "run-url=https://github.com/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID" >> $GITHUB_OUTPUT
+
+      #Will edit the comment that triggered the /test comment and add the run-url
```
Comment (RothAndrew) [inline .github/workflows/test-command.yml:128]:
```suggestion
      #Will update the comment that triggered the /test comment and add the run-url
```

File: .github/workflows/test-command.yml [hunk 4]
```diff
@@ -105,6 +121,22 @@ jobs:
...
+      - name: Create comment
```
Comment (RothAndrew) [inline .github/workflows/test-command.yml:129]:
```suggestion
      - name: Update comment
```

**Review by RothAndrew — CHANGES_REQUESTED** [review]

**Review by RothAndrew — APPROVED** [review]

**Pull request merged**